### PR TITLE
Add instructions of how to setup K8S in README

### PR DESCRIPTION
**Pull request by lianhao**
## Description

Add instructions of how to setup K8S in README

## Issues

n/a

## Dependencies

n/a

## Tests

n/a

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,11 +13,27 @@ The GenAIInfra repository is organized under four main directories, which includ
 | `manifests`         | Manifests for deploying [GenAIExamples](https://github.com/opea-project/GenAIExamples) on Kubernetes and on Docker Compose. |
 | `scripts`           | Scripts for testing, tools for facilitate OPEA deployment, and etc.                                                         |
 
-## Usages
+## Prerequisite
+
+GenAIInfra uses Kubernetes as the cloud native infrastructure. Please follow the following steps to prepare the Kubernetes environment.
+
+### Setup Kubernetes cluster
+
```
Comment (yongfengdu) [inline README.md:21]: Suggest to use or add the kubespray link, which is used for production ready k8s deployment with simple and clear instructions.
https://github.com/kubernetes-sigs/kubespray/blob/master/README.md

File: README.md [hunk 2]
```diff
@@ -13,11 +13,27 @@ The GenAIInfra repository is organized under four main directories, which includ
...
 
-### Prerequisites
+The following steps are optional. They're only required if you want to run the workloads on Intel Gaudi product.
 
-GenAIInfra uses [Kubernetes](https://kubernetes.io/) as the cloud native infrastructure. You will need the access to either managed kubernetes services (e.g., EKS,ASK,GKE and etc) or self managed kubernetes.
+1. Please check the [support matrix](https://docs.habana.ai/en/latest/Support_Matrix/Support_Matrix.html) to make sure that environment meets the requirements.
+
+2. [Install Intel Gaudi software stack](https://docs.habana.ai/en/latest/Installation_Guide/Bare_Metal_Fresh_OS.html#driver-fw-install-bare).
+
+3. [Install and setup container runtime](https://docs.habana.ai/en/latest/Installation_Guide/Bare_Metal_Fresh_OS.html#set-up-container-usage), based on the container runtime used by Kubernetes.
```
Comment (yongfengdu) [inline README.md:32]: Suggest to copy the instructions of configuring containerd. The original link default is docker setup, someone may have issues to find the correct place.
sudo tee /etc/containerd/config.toml <<EOF
disabled_plugins = []
version = 2

[plugins]
  [plugins."io.containerd.grpc.v1.cri"]
    [plugins."io.containerd.grpc.v1.cri".containerd]
      default_runtime_name = "habana"
      [plugins."io.containerd.grpc.v1.cri".containerd.runtimes]
        [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.habana]
          runtime_type = "io.containerd.runc.v2"
          [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.habana.options]
            BinaryName = "/usr/bin/habana-container-runtime"
  [plugins."io.containerd.runtime.v1.linux"]
    runtime = "habana-container-runtime"
EOF

File: README.md (hunk 1 above)
Comment (lianhao) [reply README.md:21]: done

File: README.md (hunk 2 above)
Comment (lianhao) [reply README.md:36]: K8S now supports both containerd and docker-ce container runtime. I've verified on ubuntu 22.04 for both of them.

I've added the NOTE paragraph for users to configure the right container runtime down

**Comment by lianhao** [comment]
I've verified the k8s installation steps for both xeon/gaudi on ubuntu 22.04 OS.

**Review by jfding — APPROVED** [review]

**Review by chensuyue — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -13,11 +13,33 @@ The GenAIInfra repository is organized under four main directories, which includ
...
-## Usages
+## Prerequisite
+
+GenAIInfra uses Kubernetes as the cloud native infrastructure. Please follow the following steps to prepare the Kubernetes environment.
+
+### Setup Kubernetes cluster
+
+Please follow [Kubernetes official setup guide](https://kubernetes.io/docs/setup/) to setup the Kubernetes itself. We recommend to use Kubernetes with version >= 1.27.
+
+There are different methods to setup Kubernetes procution cluster, such as [kubeadm](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/), [kubespray](https://kubespray.io/), and [more](https://kubernetes.io/docs/setup/production-environment/tools/).
```
Comment (mkbhanda) [inline README.md:24]: s/procution/production

File: README.md [hunk 4]
```diff
@@ -13,11 +13,33 @@ The GenAIInfra repository is organized under four main directories, which includ
...
 | `scripts`           | Scripts for testing, tools for facilitate OPEA deployment, and etc.                                                         |
 
-## Usages
+## Prerequisite
+
+GenAIInfra uses Kubernetes as the cloud native infrastructure. Please follow the following steps to prepare the Kubernetes environment.
+
+### Setup Kubernetes cluster
+
+Please follow [Kubernetes official setup guide](https://kubernetes.io/docs/setup/) to setup the Kubernetes itself. We recommend to use Kubernetes with version >= 1.27.
```
Comment (mkbhanda) [inline README.md:22]: s/s/the Kubernetes itself/Kubernetes.

File: README.md [hunk 5]
```diff
@@ -13,11 +13,33 @@ The GenAIInfra repository is organized under four main directories, which includ
 | `manifests`         | Manifests for deploying [GenAIExamples](https://github.com/opea-project/GenAIExamples) on Kubernetes and on Docker Compose. |
 | `scripts`           | Scripts for testing, tools for facilitate OPEA deployment, and etc.                                                         |
 
-## Usages
+## Prerequisite
+
+GenAIInfra uses Kubernetes as the cloud native infrastructure. Please follow the following steps to prepare the Kubernetes environment.
```
Comment (mkbhanda) [inline README.md:18]: s/following/below/

**Review by mkbhanda — COMMENTED** [review]
@lianhao single typo, production spelling, if you do not mind.
I tacked on the others only because of the above.

File: README.md (hunk 3 above)
Comment (lianhao) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (lianhao) [reply README.md]: done

File: README.md (hunk 5 above)
Comment (lianhao) [reply README.md]: done

**Comment by lianhao** [comment]
> @lianhao single typo, production spelling, if you do not mind. I tacked on the others only because of the above.

Thanks for catching those @mkbhanda . I've corrected them.

**Pull request merged**